src/core/algorithms/md/hymd/validator.cpp

```diff
@@ -431,7 +431,7 @@ void Validator::MakeWorkingAndRecs(lattice::ValidationInfo const& info,
     std::vector<ColumnClassifierValueId> const removed_ccv_ids =
             lattice_->RemoveExisting(lhs, indices);
     std::vector<ColumnClassifierValueId> const interestingness_ccv_ids =
-            lattice_->GetInterestingnessCCVIds(lhs, indices);
+            lattice_->GetInterestingnessCCVIds(lhs, indices, removed_ccv_ids);
```

Comment:
I just figured out that you can decrement every element of `removed_ccv_ids`, pass that, then increment it back. Rename the `Raise...` and `Get...` methods' parameter from `removed_ccv_ids` to `ccv_id_bounds` too.